app/services/report_pdf_builder.rb

```diff
@@ -44,6 +44,8 @@ def run
       combined_pdf << CombinePDF.parse(pdf)
     end
 
+    length = combined_pdf.pages.length
+    combined_pdf.number_pages number_format: " %d / #{length} ", number_location: [:bottom_right]
```

Comment:
@luigi If you ever need to number PDF pages after you've combined them with CombinePDF, its easy.